sdk/core/azure-core/CHANGELOG.md

```diff
@@ -1,25 +1,45 @@
 # Release History
 
-## 1.8.0-beta.4 (Unreleased)
+## 1.8.0 (2023-02-02)
 
 ### Features Added
```

Comment:
see my other PR for tracing, there are changes to the Core CHANGELOG.md that need to be integrated in this PR.

Reply:
Let's move those changes to Core's changelog from your PR to this PR.